traffic/traffic_info.hpp

```diff
@@ -43,8 +44,11 @@ class TrafficInfo
     uint16_t m_idx : 15;
 
     // The direction of the segment.
-    // todo(@m) Write up what the values 0 and 1 mean specifically.
     uint8_t m_dir : 1;
+
+    // m_dir can be kDefaultDirection or kReverseDirection.
+    static int constexpr DefaultDirection() { return 0; }
```

Comment:
Why functions? Why not constants? Just declare:
```c++
static int constexpr kForwardDirection = 0;
static int constexpr kReversedDirection = 1;
```
And move them to the top of the public section (or after declaration of RoadSegmentId).